definitions/variable/emissions/carbon-removal.yaml

```diff
@@ -2,8 +2,71 @@
     description: Gross removals of carbon dioxide (CO2) from atmospheric origin
       or biomass through deliberate human activities
     unit: Mt CO2/yr
-    notes: Values should be reported as positive numbers
-- Carbon Removal|{Carbon-Removal Option}:
-    description: Gross removals of carbon dioxide (CO2) by {Carbon-Removal Option}
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
+- Carbon Removal|Geological Storage:
+    description: Gross removals of carbon dioxide (CO2) from atmospheric origin
+      or biomass through geological storage
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+- Carbon Removal|Geological Storage|{Geogological Storage Source}:
+    description: Gross removals of carbon dioxide (CO2) through {Geogological Storage Source}
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
+- Carbon Removal|Land Use:
+    description: Gross removals of carbon dioxide (CO2) through land-based sinks
+      (e.g., afforestation, soil carbon enhancement, biochar)
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+- Carbon Removal|Land Use|{Land Removal Option}:
+    description: Gross removals of carbon dioxide (CO2) through {Land Removal Option}
+    unit: Mt CO2/yr
+    tier: 2
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
+- Carbon Removal|Long-Lived Materials:
```

Comment:
I'm a bit torn here. What about materials with a lifetime of 5 years, e.g. plastics? They would have to be reported as removals in one timestep, and emissions in the next. So would rather stick with only materials here, even though I see the appeal of adding "long-lived"